client/my-sites/plugins/plugins-discovery-page/upgrade-nudge.jsx

```diff
@@ -26,6 +28,7 @@ import { getSelectedSite } from 'calypso/state/ui/selectors';
 const UpgradeNudge = ( { siteSlug, paidPlugins } ) => {
 	const selectedSite = useSelector( getSelectedSite );
 	const sitePlan = useSelector( ( state ) => getSitePlan( state, selectedSite?.ID ) );
+	const isEcommerceTrial = sitePlan?.product_slug === PLAN_ECOMMERCE_TRIAL_MONTHLY;
```

Comment:
Nit: it would probably be good to implement an `isEcommerceTrial()` helper function in `calypso-products`, but that doesn't block this PR, as it was already missing.